backend/hubspot/hubspot.go

```diff
@@ -382,9 +382,7 @@ func (h *HubspotApi) CreateContactCompanyAssociation(ctx context.Context, adminI
 
 func (h *HubspotApi) CreateContactCompanyAssociationImpl(ctx context.Context, adminID int, workspaceID int) error {
 	key := fmt.Sprintf("hubspot-association-%d-%d", adminID, workspaceID)
-	if err := h.redisClient.AcquireLock(ctx, key, ClientSideCreationPollInterval); err != nil {
-		return e.New("failed to acquire hubspot association lock")
-	} else {
+	if acquired := h.redisClient.AcquireLock(ctx, key, ClientSideCreationPollInterval); acquired {
```

Comment:
Not blocking since pre-existing - I was a little confused by the the term `PollInterval` part of the variable here since it seems to be used as a timeout value instead. Just want to make sure we want to be used here in sync with its other usages.

i.e. if we change the interval in `pollHubspot` that we are okay with it changing here as well

Reply:
great point... this should just be a different short time constant and has nothing to do with the poll interval. updating!